docs/simplesamlphp-upgrade-notes-2.0.md

```diff
@@ -144,3 +145,53 @@ $x = \SimpleSAML\Utils\Arrays::arrayize($someVar)
 
 [1]: https://github.com/simplesamlphp/simplesamlphp/wiki/Migrating-translations-(pre-migration)
 [2]: https://github.com/simplesamlphp/simplesamlphp/wiki/Twig:-Migrating-templates
+
+## Upgrading and EntityIDs
+
+If you still have your 1.x installation available, the entityID you
+are using for your SP and IdP should be available in
+module.php/core/frontpage_federation.php location on your
+SimpleSAMLphp server.
```

Comment:
I would add a new paragraph:

"For a service provider, if it was set as auto-generated in 1.19, it will likely have the form of `https://<your hostname>/simplesaml/module.php/saml/sp/metadata.php/default-sp`."

Reply:
adding.